modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -36,57 +36,44 @@ Built-in dashboards include various pages to view different types of charts:
 [%header,cols="1,4"]
 |===
 | Built-In Dashboards | Charts
-| *Overview* a|
+| <<overview-charts>> a|
 
 * Inbound - Total Requests by Response Type
 * Inbound - Average Response Time
-* Inbound - Mule messages
+* Mule Messages
 * Outbound - Total Requests by Response Type
 * Outbound - Average Response Time
 * JVM - CPU utilization
 * JVM - Heap memory utilization
```

Comment:
UX displays "JVM - Heap Used"

@myuin-ms: do we update the doc or the UX for this one?

Reply:
```suggestion
* JVM - Heap Used
```